### PR TITLE
Module_1_core_concepts

### PR DESCRIPTION
**Pull request by VolhaChubakova**

### CODE REVIEW

**Comment by supercilium** [comment]
it's better to follow cra folder structure (not create sub-directories in the project folder) or add /module_1/node_modules in .gitignore

I would also recommend to setup a prettier for code styling https://medium.com/@frontend-newbie/integrating-prettier-with-eslint-for-your-create-react-app-in-vscode-153ebe89c1a2

File: module_1/src/SearchForm/SearchForm.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            className='SearchForm-input'
+            placeholder="What do you want to watch"
+            value={query}
+            onChange = {handleChange}
+            onKeyDown = {(e) => {
+                if (e.code === 'Enter') {
+                    props.onSearch(query)
+                }
+            }}
+        ></input>
```
Comment (supercilium) [inline module_1/src/SearchForm/SearchForm.js]: elements without content can be self-closed <input {...props} **/>**

File: module_1/src/GenreSelect/GenreSelect.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import { IoMdArrowDropdown } from 'react-icons/io';
+
+
+function GenreSelect (props) {
+    const [isOpened, setOpened] = useState(true);
+    const [rotateIcon, setRotateIcon] = useState(false);
+    const [currentlySelectedArr, setCurrentlySelectedArr] = useState(props.currentlySelectedArr);
+    
+    const rotate = rotateIcon ? "rotate(180deg)" : "rotate(0)";
+    const iconStyles = { color: "#F65261", fontSize: "1.5em", position: 'absolute', top: '12px', right: '16px', transform: rotate  };
```
Comment (supercilium) [inline module_1/src/GenreSelect/GenreSelect.js]: you can manage this on css class level, the classnames library can make easier operation with them
https://www.npmjs.com/package/classnames

**Comment by VolhaChubakova** [comment]
Thanks a lot for the feedback. I have added necessary changes.

**Pull request merged**